src/Aspire.Dashboard/Components/Pages/ConsoleLogs.razor.cs

```diff
@@ -308,10 +312,11 @@ private void UpdateMenuButtons()
         }
     }
 
-    private async Task ToggleTimestamp(bool hideTimestamp)
+    private async Task ToggleTimestampAsync(bool hideTimestamp)
     {
-        PageViewModel.HideTimestamp = hideTimestamp;
-        await this.AfterViewModelChangedAsync(_contentLayout, waitToApplyMobileChange: false);
+        await SessionStorage.SetAsync(HideTimestampStorageKey, new ConsoleLogTimestampState(HideTimestamp: hideTimestamp));
+        _hideTimestamp = hideTimestamp;
+        StateHasChanged();
```

Comment:
Is this nessessary? I would have thought that it would be triggered automatically after the click event

Reply:
Surprisingly yes, it is necessary. without:
![Animation](https://github.com/user-attachments/assets/acf4bce5-ebf7-4f29-959c-193ca1e21937)